discord.bat

```diff
@@ -1,3 +1,34 @@
+@echo off
+chcp 65001 >nul
+:: 65001 - UTF-8
+
+:: Path check
+set scriptPath=%~dp0
+set "path_no_spaces=%scriptPath: =%"
+if not "%scriptPath%"=="%path_no_spaces%" (
+    echo Путь содержит пробелы. 
+    echo Пожалуйста, переместите скрипт в директорию без пробелов.
+    pause
+    exit /b
+)
+
+:: Cyrillic check
+echo %scriptPath% | findstr /r "[А-Яа-яЁё]" >nul

```

Comment:
Strange behavior. **запрет-zapret-discord-youtube** returns ErrorLevel 0, but **абв-zapret-discord-youtube** returns ErrorLevel 1

Reply:
The problem is that we work UTF-8, because of what some letters he just ignores, and does not consider for сyrillic , it can be corrected only by using encoding 1251 (Windows-1251) 
But then we will get something like this 

РљРёСЂРёР "Р "РёС†Р° - Р СѓСЃСЃРёР№ Р°Р "С "Р°РРІРёС'.

The file should be saved in the same encoding, but I suppose that GitHub will probably convert it all to UTF-8 The code will be a little bit bigger, but in theory it guarantees us 100% validation of the Cyrillic characters